crates/utils/re_video/src/demux/mod.rs

```diff
@@ -115,6 +115,27 @@ pub type GopIndex = usize;
 /// Index used for referencing into [`VideoDataDescription::samples`].
 pub type SampleIndex = usize;
 
+/// Distinguishes finate videos from video streams.
```

Comment:
```suggestion
/// Distinguishes finite videos from video streams.
```

Reply:
Not sure about the term "finite" though. Streams are also finite, it's just that we don't yet know how long they are.

Alternative suggestions:
* File
* Static
* Fixed
* Asset